sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogDetailLevel.java

```diff
@@ -34,6 +39,36 @@
      */
     BODY_AND_HEADERS;
 
+    static final String BASIC_VALUE = "basic";
+    static final String HEADERS_VALUE = "headers";
+    static final String BODY_VALUE = "body";
+    static final String BODY_AND_HEADERS_VALUE = "body_and_headers";
+    static final String BODYANDHEADERS_VALUE = "bodyandheaders";
+
+    static final HttpLogDetailLevel ENVIRONMENT_HTTP_LOG_DETAIL_LEVEL = fromConfiguration(getGlobalConfiguration());
+
+    static HttpLogDetailLevel fromConfiguration(Configuration configuration) {
+        String detailLevel = configuration.get(Configuration.PROPERTY_AZURE_HTTP_LOG_DETAIL_LEVEL);
+
+        HttpLogDetailLevel logDetailLevel;
+        if (CoreUtils.isNullOrEmpty(detailLevel)) {
```

Comment:
This can be removed if the above change is made.

Reply:
Done